src/core/worker.js

```diff
@@ -513,6 +515,45 @@ class WorkerMessageHandler {
       });
     });
 
+    handler.on("ResetNewRef", function wphSetupResetNewRef() {
+      pdfManager.pdfDocument.xref.resetNewRef();
+    });
+
+    handler.on("GetInfoForNewXRef", function wphSetupGetInfoForNewXRef() {
+      const xref = pdfManager.pdfDocument.xref;
+      if (xref.trailer) {
+        const _info = {};
```

Comment:
*Nit (here and elsewhere):* In the PDF.js library we usually use `Object.create(null)`, rather than `{}`, to prevent any possible issues if a library user have e.g. extended the `Object.prototype` incorrectly.